controllers/funding/materials-helpers.js

```diff
@@ -239,8 +240,31 @@ If you have feedback, please contact matt.andrews@biglotteryfund.org.uk.`;
     return text.trim();
 }
 
+async function injectMerchandise(req, res, next) {
+    try {
+        res.locals.availableItems = await contentApi.getMerchandise(req.i18n.getLocale());
+        next();
+    } catch (error) {
+        next(error);
+    }
+}
+
+function injectMerchandiseCustom({ locale = false, showAll = false }) {
```

Comment:
Can `injectMerchandiseCustom` and `injectMerchandise` be merged into a single method?

Reply:
Just did this!